webpack/scenes/Subscriptions/SubscriptionActions.js

```diff
@@ -88,10 +88,10 @@ export const updateQuantity = (quantities = {}) => async (dispatch) => {
 
   try {
     const { data } = await api.put(`/organizations/${orgId()}/upstream_subscriptions`, params);
-    return dispatch({
-      type: UPDATE_QUANTITY_SUCCESS,
-      response: data,
-    });
+
+    dispatch(pollTaskUntilDone(data.id, {}, POLL_TASK_INTERVAL, Number(orgId())));
```

Comment:
pollTaskUntilDone ensures that the progress modal shows up immediately rather than on the next poll that the page does naturally